### PR TITLE
Feature/basic cf setup

### LINKED ISSUE #19: Write basic setup logic for hurdles
Write basic setup code that create a new Cloud Foundry org for the test run. The tool has to make sure that no matter what happens, it will eventually delete the org at the end. This also includes a hard exit of the program. If possible, catch Ctrl-C events as well.

### PR DESCRIPTION
**Pull request by LynxPlay101**
This branch contains code to set up a basic test env for watchful on the cloud foundry. 
For the sake of testing and coming closer to the CLI, it also already contains an alpha version of the run command.

This fixes #19

### CODE REVIEW

File: pkg/logger/logger_pipeline.go [hunk 1]
```diff
@@ -71,43 +68,57 @@ func (s *SplitPipeline) Write(messages []ChannelMessage) {
...
 
-			if line < len(groupArray) {
+			if line < groupArraySize {
 				groupOutput := groupArray[line]
-				output += groupOutput + strings.Repeat(" ", s.config.CharacterPerPipe-bunt.PlainTextLength(groupOutput)) + "\u001b[0m"
+				output += groupOutput
+
+				if s.config.GroupContainer.GroupCount() > 1 {
+					emptyChars := s.config.CharacterPerPipe - bunt.PlainTextLength(groupOutput) // The prefix is already in the group output
+					output += strings.Repeat(" ", emptyChars)
```
Comment (qu1queee) [inline pkg/logger/logger_pipeline.go]: this will eventually give you a negative number, check this.

Comment (HeavyWombat) [reply pkg/logger/logger_pipeline.go]: @LynxPlay101 I did not have time to review this one here. If the other work scales down, I will have a look. Until then, what do you think about the comment made by Enrique?

**Comment by LynxPlay101** [comment]
This branch is outdated and will be reimplemented by feature/general-workflow

File: pkg/logger/logger_pipeline.go (hunk 1 above)
Comment (LynxPlay101) [reply pkg/logger/logger_pipeline.go]: Closing it was my response 😂 The new branch contains a first test push, and works fine. I could not reproduce the negative repeating numbers

**Pull request closed without merging**